conbench/templates/index.html

```diff
@@ -96,27 +96,33 @@ <h4>Recent runs</h4>
                 </tr>
                 {% endfor %}
             </tbody>
-        </table>
-    </div>
+</table>
 </div>
 
-
 {% endblock %}
 
 
 {% block scripts %}
 {{super()}}
 <script>
   var table = $('#runs').DataTable({
+      // the default default seems to be the first item in lengthMenu.
+      "lengthMenu": [ 5, 10, 50, 75, 100 ],
+      // but when pageLength is also set, then _this_ is the default.
+      "pageLength": 10,
       "order": [[0, 'desc']],
+      // https://datatables.net/reference/option/dom
+      // put `l` to the bottom
+      "dom": "frtipl",
```

Comment:
try to pronounce that!

powerful config interface: https://datatables.net/examples/basic_init/dom.html